Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue6556.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+
+
+namespace Xamarin.Forms.Controls.Issues
+{
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 6556, "Button.CornerRadius doesn't work on WPF", PlatformAffected.WPF)]
+	public class Issue6556 : TestContentPage
+	{
+		protected override void Init()
+		{
+			var sl = new StackLayout { Padding = new Size(20, 20) };
+			sl.Children.Add(new TestButton(5));
+			sl.Children.Add(new TestButton(10));
+			sl.Children.Add(new TestButton(20));
+			sl.Children.Add(new Button()
+			{
+				Text = "Round",
+				WidthRequest = 50,
+				HeightRequest = 50,
+				CornerRadius = 25,
+				BorderWidth = 5,
+				HorizontalOptions = LayoutOptions.Center
+			});
+			Content = sl;
```

Comment:
The following comment is optional, but it would be great to have something like:

```
sl.Children.Add (new Button
{
     BorderWidth = 5,
     Text = "No CornerRadius"
});
```

To test different options using CornerRadius in the same test but also validate a Button without it.